specification/fluidrelay/resource-manager/Microsoft.FluidRelay/stable/2022-04-21/fluidrelay.json

```diff
@@ -0,0 +1,1060 @@
+      ],
+      "x-adl-name": "Page<FluidRelayServer>"
+    },
+    "FluidRelayContainerList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FluidRelayContainer"
+          },
+          "description": "A sequence of FluidRelay containers.",
+          "x-adl-name": "FluidRelayContainer[]"
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A link to the next page of results, if any."
+        }
+      },
+      "description": "Paged response"
+    },
+    "FluidRelayServer": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/FluidRelayServerProperties",
+          "description": "The resource-specific properties for this resource.",
+          "readOnly": false,
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "System meta data for this resource, including creation and modification information.",
+          "readOnly": true
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The type of identity used for the resource."
+        }
+      },
+      "description": "A FluidRelay Server.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "FluidRelayContainer": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/FluidRelayContainerProperties",
+          "description": "The resource-specific properties for this resource.",
+          "readOnly": true,
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "System meta data for this resource, including creation and modification information.",
+          "readOnly": true
+        }
+      },
+      "description": "A FluidRelay Container.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "FluidRelayServerKeys": {
+      "type": "object",
+      "properties": {
+        "key1": {
+          "type": "string",
+          "format": "password",
+          "description": "The primary key for this server",
+          "readOnly": true
+        },
+        "key2": {
+          "type": "string",
+          "format": "password",
+          "description": "The secondary key for this server",
+          "readOnly": true
```

Comment:
not sure how this got missed last time , but this needs to be an x-ms-secret

Reply:
According to this: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret, "When a property is modeled as "readonly": true, then applying this extension as "x-ms-secret": true is not allowed." Are you saying that we should remove the "readOnly: true" and use "x-ms-secret: true" instead?

Reply:
Ok sorry ignore this. Since this is the response from a POST operation it is fine. I misread this as the response from a GET.